src/components/Map.js

```diff
@@ -86,7 +88,7 @@ const MyMap = withGoogleMap((props) => {
                   </div>
                   <div className="flex vertCenter infoItemWrapper">
                     {/* TODO change the icon to YEN */}
-                    <DollarIcon style={svgIconSize} />
+                    <DollarIcon style={svgIconSize} viewBox="4 4 19 19" />
```

Comment:
@bill-feng Sorry, if you could remove the comment` {/* TODO change the icon to YEN */}`, that'll be appreciated! :)